server/app/routes.py

```diff
@@ -1,12 +1,13 @@
 from flask import render_template, send_from_directory, send_file
-from app import app 
+from app import app
 # Load routes from modules in "controllers" directory
 from app.controllers import *
+from app.utils import get_footer
 
 @app.route('/')
 @app.route('/index')
 def index():
-    return render_template("index.html")
+    return render_template("index.html", footer = get_footer())
```

Comment:
Controller should return pure data, not HTML.

Reply:
The code is now updated. It returns pure data and returns 2 things: SHA of the last git commit and the version, where the file was updated. The former tells us what version is running, the latter is an indicator whether the update routine is working or not.